chain/chain/src/chain.rs

```diff
@@ -1540,7 +1540,7 @@ impl Chain {
         for (_, receipt_proofs) in receipt_proofs_by_shard_id.iter_mut() {
             let mut slice = [0u8; 32];
             slice.copy_from_slice(block.hash().as_ref());
-            let mut rng: StdRng = SeedableRng::from_seed(slice);
+            let mut rng: ChaCha20Rng = SeedableRng::from_seed(slice);
             receipt_proofs.shuffle(&mut rng);
         }
```

Comment:
Could we move this loop into separate function and then test the output of the function?